static/scss/search-page.scss

```diff
@@ -56,6 +56,11 @@
       padding: 0 10px 0 23px;
     }
 
+    .sk-hierarchical-menu-list {
+      max-height: 150px;
+      overflow: auto;
```

Comment:
just one comment: it might be better to use `overflow: scroll;` to just be a little more intentional about the behavior we want.